local-files/trino-entrypoint.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
 
 export PATH="${JAVA_HOME}/bin:${PATH}"
-usr/lib/trino/bin/launcher run
+usr/lib/trino/bin/launcher stop
+usr/lib/trino/bin/launcher start
+sleep 2
```

Comment:
Does it make sense to check if the file exists in a loop? Does `sleep` guarantee that enough time will have passed? Off the top of my head it would look something like this:
```sh
while [ ! -f /data/trino/var/log/server.log ]; do
    sleep 1
done
```

Reply:
Good point. Thank you, I will add some logic to check for the existence of the file.

Reply:
Updated.